data/derived/litter/turnover/decay_parameters.csv

```diff
@@ -0,0 +1,8 @@
+Parameter,value,.lower,.upper,.width,.point,.interval
+fM,0.6282879158153394,0.5531372802083837,0.7242938524970657,0.9,median,qi
+km,0.022175290256387016,0.014915193955037737,0.031562162918338556,0.9,median,qi
+ks,0.0023939156570189964,0.0015150843916721542,0.0036986142855893285,0.9,median,qi
+kw,8.92160792597574e-5,5.139449306745886e-5,1.5122042745491823e-4,0.9,median,qi
+r,-0.38318229910718593,-0.8263192925616477,-0.1791438620993433,0.9,median,qi
+sN,1.8078330445505075e-4,1.0328504549546988e-4,2.915114255848605e-4,0.9,median,qi
+sP,8.68447800836268e-6,5.537780078959081e-6,1.2695950357673214e-5,0.9,median,qi
```

Comment:
These are also a lot lower (which suggests that the split between metabolic and structural litter is less sensitive to chemistry). I guess we are looking at a consistent pattern of the SAFE specific decay being less sensitive to chemistry than the global parameters Fatichi et al (2019) estimated. Which feels sane to me